build/BranchInfo.props

```diff
@@ -1,8 +1,25 @@
 <Project>
   <PropertyGroup>
+    <StableProjects>
+        Microsoft.ML;
+        Microsoft.ML.CpuMath;
```

Comment:
Are we going to add the other assemblies?

Reply:
Yes. Seems like only adding them here won't fix their versioning when they're used inside another package